lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -183,6 +179,31 @@ def print_ccache_contents(ccache, key: nil)
             presenter = Rex::Proto::Kerberos::CredentialCache::Krb5CcachePresenter.new(ccache)
             print_status presenter.present(key: key)
           end
+
+          # Build a loot info string that can later be used in a lookup.
+          #
+          # @param [Hash] options
+          # @option options [String] :realm the realm of the ticket (optional)
+          # @option options [String] :server the service name of the ticket (optional)
+          # @option options [String] :client the client username of the ticket (optional)
+          # @option options [Boolean] :valid whether or not the ticket is valid, defaults to true (optional)
+          # @return [String] the info string
+          def self.loot_info(options = {})
+            info = []
+
+            info << '[invalid]' if !options.fetch(:valid, true)
```

Comment:
This isn't used now, but in the future, we can use it to update info of tickets that we know are invalid for whatever reason.